src/frequenz/sdk/timeseries/_formula_engine/__init__.py

```diff
@@ -4,11 +4,13 @@
 """A formula engine for applying formulas."""
 from ._formula_engine import FormulaEngine, FormulaEngine3Phase
 from ._formula_engine_pool import FormulaEnginePool
+from ._formula_formatter import format_formula
 from ._resampled_formula_builder import ResampledFormulaBuilder
 
 __all__ = [
     "FormulaEngine",
     "FormulaEngine3Phase",
     "FormulaEnginePool",
     "ResampledFormulaBuilder",
+    "format_formula",
```

Comment:
```suggestion
```

There is no need to export here, these are all private modules and internal imports should be done using the original location to avoid potential circular dependencies.